src/components/Header/Header.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { PureHeader, Header } from './Header';
+
+describe('PureHeader', () => {
+  it('renders', () => {
+    const menu = [{ label: "testlabel1", path: "/testlabel1" }]
+    const data = { site: { siteMetadata: { title: 'Test Title', menu } } };
+    const { container } = render(<PureHeader data={data} />);
+    expect(container).not.toBeNull();
```

Comment:
Doing a null check here and then subsequently doing a test on the contents of the PureHeader renders this test redundant. All of the subsequent tests on the PureHeader prove that the contents aren't null.